src/MockBot.Api/Controllers/ChatController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+
+namespace MockBot.Api.Controllers
+{
+    [Route("api/chats")]
+    [ApiController]
+    public class ChatController : ControllerBase
+    {
+        private readonly IChatService _chatService;
+
+        public ChatController(IChatService chatService)
+        {
+            _chatService = chatService;
+        }
+
+        [HttpGet]
+        public OkObjectResult FindAll()
+        {
+            var chats = _chatService.FindAll();
+
+            return Ok(chats);
+        }
+
+        [HttpGet("{id}")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Chat))]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public ActionResult Get(Guid id)
```

Comment:
Reference to comment on line 27:
```suggestion
        public ActionResult<Chat> Get(Guid id)
```